python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -281,6 +282,42 @@ def run_individual(self, number, repeat=1, min_repeat_ms=0):
         ret = self._run_individual(number, repeat, min_repeat_ms)
         return ret.strip(",").split(",") if ret else []
 
+    def run_individual_node(self, index, number, repeat=1, min_repeat_ms=0):
+        """Benchmark a single node in the serialized graph.
+
+        Parameters
+        ----------
+        index : int
+            The index of the node, see `self.debug_datum.get_graph_nodes`
+
+        number: int
+            The number of times to run the node to get a benchmark result.
+
+        repeat: int
+            The number of times to benchmark the nodes.
+
+        min_repeat_ms: int
+            The minimum consecutive runtime of the node for a benchmark result.
+
+        Returns
+        -------
+        A list of dimensions `number` x `repeat` each one the runtime of the node
```

Comment:
I don't really understand this message. Also, would it make sense to return an array of `BenchmarkResult` to match `time_evaluator`?

Reply:
Basically if have like 3 repeats of 3 numbers it would return a 3x3 array/list.

arr[0][1] would be the first repeat in the second number, arr[1][2] would be the repeat 2, number 3, etc.

I think BenchmarkResult is better though since it seems to store the sequence of all float results anyway?

Reply:
`BenchmarkResult` is consistent with other interfaces, so using that would be best.

Reply:
Done